### PR TITLE
[ch19908] Add access level data to user endpoints

### PR DESCRIPTION
**Pull request by nmashton**
As part of Core UI's work adding basic roles and permissions to the Management app, we're adding a notion of "access levels" controlled via a new table in SQL Server, `AccessLevel`, that's linked to users on a per-network basis. To work with this new kind of user data, we need to augment the User endpoints with access level data.

This PR adds `AccessLevel` as part of the readable User data and adds optional `AccessLevel` parameters to the Create and Update endpoints. It also enumerates the valid values in a schema.

**These changes are not yet in prod.** (The PR in api-proxy is https://github.com/adzerk/api-proxy/pull/303 .)

### CODE REVIEW

**Comment by daveyarwood** [comment]
@CrshOverride @calliebensel I wanted to call your attention to this. It's looking like we will ship this change to production during the maintenance window this coming Sunday.

**Comment by CrshOverride** [comment]
@daveyarwood Thanks for the heads up. I'll review now and merge Monday after it's live.

File: management/schemas/user.yaml [hunk 1]
```diff
@@ -34,3 +36,10 @@ schemas:
         type: array
         items:
           $ref: '#/schemas/User'
+
+  AccessLevel:
+    type: string
+    enum:
+      - read
+      - edit
+      - admn
```
Comment (CrshOverride) [inline management/schemas/user.yaml:45]: Is this a typo? Not sure if it should really be `admn` or `admin` (spelled out).

Comment (daveyarwood) [reply management/schemas/user.yaml:45]: `admn` is correct. We decided on a 4-character string for the database column, and in the API we are keeping things simple and just using that value directly.

**Review by CrshOverride — APPROVED** [review]

**Pull request merged**